### PR TITLE
User activity tracker using redis

### PR DESCRIPTION
**Pull request by Rupeshiya**
---
name: Pull Request template
about: Describe the PR.
title: ''
labels: ''
assignees: ''

---

### **Problem**
Github Issue Number: #150 
As of now, there is no backend implemented to track user activity on to the platform _

### **Solution of problem**
Implement the backend using redis to track user activity on to the platform

### **Type of Change**
[ ] Bug fix
[&check;] New Feature
[ ] Development of UI/UX prototypes
[ ] Small refactor
[ ] Change in Documentation

### **Checklist**
[&check;] My code follows the same style as the codebase
[ ] My Code change requires a change in documentation
[ ] I have updated the Readme accordingly
[&check;] I made PR against **development branch**
[&check;] I have run the test cases locally and it's passing.
[ ] I have squashed my commits

### CODE REVIEW

File: app/controllers/event.js [hunk 1]
```diff
@@ -163,7 +168,11 @@ module.exports = {
         notification.content = `Event ${deleteEvent.eventName} is deleted!`
         notification.tag = 'Deleted'
         notificationHelper.addToNotificationForAll(req, res, notification, next)
-        return res.status(HttpStatus.OK).json({ deleteEvent: deleteEvent, message: 'Deleted the event' })
+        activityTracker.addToRedis(req, res, next, collectionTypes.EVENT, event._id)
```
Comment (kmehant) [inline app/controllers/event.js:171]: @Rupeshiya 
I am little bit curious about tracking delete actions. I assume that once the event (or any deleted entity) is deleted, how does the `event._id` would play a role? How are you going to know which event (or any deleted entity) as `_id` becomes a dangling reference?

Comment (Rupeshiya) [reply app/controllers/event.js:171]: @kmehant For better understanding go ahead fetch locally and run tests. 🎉

Comment (Rupeshiya) [reply app/controllers/event.js]: Thanks for pointing out. ✌️

**Comment by Rupeshiya** [comment]
@vaibhavdaren Please review

**Pull request merged**